back/engines/commercial/public_api/config/routes.rb

```diff
@@ -42,6 +41,9 @@
       end
     end
 
+    # Ideas with create and update actions (separate from the with_options block)
+    resources :ideas, only: %i[index show create update], concerns: :deleted_items
```

Comment:
Fine, but best to move this next to the other resources call already outside of the block on line 20